vrk-kpa/suomidigi
# SUOPA-664 Author taxonomy

To test: 
- `make fresh`
- Go to https://suomidigi.docker.sh/admin/structure/taxonomy/manage/author/overview
- Edit any of the terms and check that there are the same fields as there is in profile edit page (https://suomidigi.docker.sh/user and click edit)
- As a bonus, there's a question "Does author have user profile?". Try to click yes and see if there's any profiles available
